src/utils/DecoyDatabase.cpp

```diff
@@ -99,6 +100,8 @@ class TOPPDecoyDatabase :
     registerFlag_("only_decoy", "Write only decoy proteins to the output database instead of a combined database.", false);
     registerStringOption_("method", "<enum>", "reverse", "Method by which decoy sequences are generated from target sequences.", false);
     setValidStrings_("method", ListUtils::create<String>("reverse,shuffle"));
+    registerStringOption_("type", "<enum>", "peptide", "Type of sequence.", false);
```

Comment:
Please use "protein" instead of "peptide" (since at this point the sequences aren't digested), both here and later in the enum.

Reply:
Also extend the documentation: "RNA sequences may contain modification codes, which will be handled correctly if this is set to 'RNA'."